okhttp-clients/src/main/java/com/palantir/conjure/java/okhttp/HostEventsSink.java

```diff
@@ -0,0 +1,14 @@
+/*
+ * (c) Copyright 2018 Palantir Technologies Inc. All rights reserved.
+ */
+
+package com.palantir.conjure.java.okhttp;
+
+/**
+ * A destination for recording host events.
```

Comment:
Could we add an explanation for why this exists? I know we didn't have anything previously either, but I think this is an opportunity to be more readable than before!